### ISSUE TITLE
[feature]: Accept raw file as body content type

### ISSUE THREAD

**Issue by mahagr** [hoppscotch/hoppscotch#2320]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

When you select request body content type, there is no option for raw data.

This option should be similar to multipart/form-data. but instead of accepting multiple name/value pairs, it should have just a single file upload field. This data would then be sent in the body just as it is.

### Why should this be worked on?

PHP does not have support for multipart/form-data for POST, PATCH, etc (as they think it's against HTTP specs). 

Instead, they recommend sending the files as a raw stream of data:
https://www.php.net/manual/en/features.file-upload.put-method.php

Unfortunately, Hoppschotch does not support this method.

This simple feature would be a great addition to Hoppschotch. :)

**Comment by muraterzioglu** [comment]
I also agree with that request, on Next.js if you want the send request with body content your body must be RAW.

DTO;
```ts
export class Detail {
    @IsString()
    @IsNotEmpty()
    challenge: string;
}
```

Controller;
```ts
  @Get(`${pathPrefix}detail`)
  detail(@Body() detail: Detail) {
    return this.service.detail(detail);
  }
```

Postman supports that feature, but postwoman don't :)

**Comment by gruessung** [comment]
+1

**Comment by frank-villanueva** [comment]
+2
